### PR TITLE
[EN-707] Add blacklist file option to opcache config

### PR DESCRIPTION
**Pull request by tjoelsson**

### CODE REVIEW

**Comment by tjoelsson** [comment]
@juwai/devops thumbs please

**Comment by stuartwan** [comment]
May I know what is the problem we are trying to solve here?

**Comment by wenzong** [comment]
:+1: 

@imlazyone we are using opcache, some file we don't want to cache it, so we want to add it in blacklist

currently we don't have such variable in this role

File: templates/opcache.ini.j2 [hunk 1]
```diff
@@ -11,3 +11,4 @@ opcache.max_wasted_percentage={{ php_opcache_max_wasted_percentage }}
 opcache.validate_timestamps={{ php_opcache_validate_timestamps }}
 opcache.revalidate_freq={{ php_opcache_revalidate_freq }}
 opcache.max_file_size={{ php_opcache_max_file_size }}
+opcache.blacklist_filename={{ php_opcache_blacklist_filename }}
```
Comment (wenzong) [inline templates/opcache.ini.j2]: ? 

```
{% if php_opcache_blacklist_filename != '' %}
...
{% endif %}
```

do you think it's better?

Comment (stuartwan) [inline templates/opcache.ini.j2]: ? Also better to have new line in the end of a file?

**Comment by stuartwan** [comment]
:+1:

File: templates/opcache.ini.j2 (hunk 1 above)
Comment (tjoelsson) [inline templates/opcache.ini.j2]: Actually having it empty is not a problem, but sure, I'll add the condition.

**Pull request merged**